src/components/films/components/filmsHeader/FilmsSort.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { useState } from "react";
+import { Sorting } from "./filmsHeader.styled";
+
+const FilmsSortC = () => {
+  const [ sortByValue, setSortByValue ] = useState('release date');
+
+  const handleChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
+    setSortByValue(e.target.value)
+  };
+
+  return(
+    <Sorting>
+      <label>
+        Sort by
+        <select value={sortByValue} onChange={handleChange}>
+          <option value="release date">release date</option>
```

Comment:
You'd be much better off with options populated from props, rather than hardcoded statically.